dask/bag/tests/test_bag.py

```diff
@@ -602,6 +602,7 @@ def test_can_use_dict_to_make_concrete():
 
 @pytest.mark.slow
 @pytest.mark.network
+@pytest.mark.skip(reason="Hangs")
```

Comment:
I'm not sure why, but when I ran `pytest` locally with `-n 2`, one of the works got stuck on this test.  The test suite finishes when I skip it locally, we'll see about Travis.